alpha/hal.cpp

```diff
@@ -116,6 +116,14 @@ uint32_t _TS_HAL::random_get(uint32_t min, uint32_t max)
 //
 // LCD
 //
+void _TS_HAL::fillScreen(uint16_t color)
```

Comment:
whats this for? blanking the screen?

Reply:
yep, added this function to the HAL to help with that, if that's a design choice that we want, it is a very loud signal, not sure if we want to do this or perhaps just a small uploading text/logo on the main screen, which will be more subtle.

Reply:
its fine, uploading is a blocking task